index.pt.Rmd

```diff
@@ -2,13 +2,8 @@
 
 <a href="https://doi.org/10.5281/zenodo.2553043"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.2553043.svg?branch=master" alt="DOI"></a>
 
-<a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/3.0/us/"><img alt="Licença Creative Commons" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-sa/3.0/us/88x31.png" /></a><br /> Este trabalho está licenciado com uma licença [Creative Commons Attribution-NonCommercial-ShareAlike 3.0 United States License](https://creativecommons.org/licenses/by-nc-sa/3.0/us/). Utilize o [Zenodo DOI](https://doi.org/10.5281/zenodo.2553043) para citar esta obra.
-
-```{r}
-#| echo: false
-#| results: 'asis' 
-#| warning: false
-source(file.path("scripts", "zenodo.R"), local = knitr::knit_global())
+<a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/3.0/us/"><img alt="Licença Creative Commons" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-sa/3.0/us/88x31.png" /></a><br /> Este trabalho está licenciado com uma licença [Creative Commons Attribution-NonCommercial-ShareAlike 3.0 United States License](https://creativecommons.org/licenses/by-nc-sa/3.0/us/).Consulte [o DOI do Zenodo](https://doi.org/10.5281/zenodo.2553043) da versão original e [o DOI da tradução](https://zenodo.org/doi/10.5281/zenodo.10797248) para saber como citá-las.
```

Comment:
The translation DOI points to the Spanish Version, is that correct?
```suggestion
<a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/3.0/us/"><img alt="Licença Creative Commons" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-sa/3.0/us/88x31.png" /></a><br /> Este trabalho está licenciado com uma licença [Creative Commons Attribution-NonCommercial-ShareAlike 3.0 United States License](https://creativecommons.org/licenses/by-nc-sa/3.0/us/). Consulte [o DOI do Zenodo](https://doi.org/10.5281/zenodo.2553043) da versão original e [o DOI da tradução](https://zenodo.org/doi/10.5281/zenodo.10797248) para saber como citá-las.
```

Reply:
Yes and no: yes as a placeholder, no for the long term. I have opened https://github.com/ropensci/dev_guide/issues/966 to make sure I don't forget to update it. Thanks!!